src/features/home/components/modules/exclusivity/ExclusivityExternalLink.tsx

```diff
@@ -44,7 +44,9 @@ const UnmemoizedExclusivityExternalLink = ({
 
 export const ExclusivityExternalLink = memo(UnmemoizedExclusivityExternalLink)
 
-const StyledTouchableLink = styled(TouchableLink)<{ isFocus?: boolean }>(({ theme, isFocus }) => ({
+const StyledTouchableLink = styled((props) => <ExternalTouchableLink {...props} />)<{
+  isFocus?: boolean
+}>(({ theme, isFocus }) => ({
```

Comment:
Pourquoi pas juste ça ?
```suggestion
const StyledTouchableLink = styled(ExternalTouchableLink)<{
  isFocus?: boolean
}>(({ theme, isFocus }) => ({
```

Reply:
C'est un bug assez bizarre, avec les styledComponent, mais en gros ça fait planter jest et la seule solution qui fonctionne c'est de faire ça... Je l'avais vu sur ce stack overflow : https://stackoverflow.com/questions/71876240/jest-cannot-create-styled-component-for-component-undefined

Reply:
il faudrait le documenter qq part sur Notion ou ailleurs (un message sur dev_mobiles ? ). C'est forcément un bug que vont rencontrer d'autres personnes

Reply:
Si tu définis ton `ExternalTouchableLink` comme une fonction (au lieu d'une const) ça devrait régler le problème
```
export function ExternalTouchableLink({
  externalNav,
  ...rest
}: FunctionComponent<ExternalTouchableLinkProps>)
```